module-utils/utility/Utils.hpp

```diff
@@ -45,6 +45,11 @@ namespace utils
         return !s.empty() && std::find_if(s.begin(), s.end(), [](char c) { return !std::isdigit(c); }) == s.end();
     }
 
+    static inline bool is_phone_number(const std::string &s)
+    {
+        return s.find_first_not_of("+#0123456789") == std::string::npos;
```

Comment:
and what if you paste +48 123 456 789, will it work?

I do believe that there is function in libphone number which handles all similar cases ;)

Reply:
hmm question is if we support spaces in Phone numbers ? to be honest I am not sure will check it. Also will try to look for that function.

Reply:
Checked -> there is no valid simple method in libphonenumber to verify that case. Added space to support presented case.